[2023-10-22T22:06:10.233+0300] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dm1trybu.train_rf scheduled__2023-10-21T09:45:00+00:00 [queued]>
[2023-10-22T22:06:10.239+0300] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dm1trybu.train_rf scheduled__2023-10-21T09:45:00+00:00 [queued]>
[2023-10-22T22:06:10.239+0300] {taskinstance.py:1359} INFO - Starting attempt 7 of 10
[2023-10-22T22:06:10.253+0300] {taskinstance.py:1380} INFO - Executing <Task(PythonOperator): train_rf> on 2023-10-21 09:45:00+00:00
[2023-10-22T22:06:10.259+0300] {standard_task_runner.py:57} INFO - Started process 228221 to run task
[2023-10-22T22:06:10.269+0300] {standard_task_runner.py:84} INFO - Running: ['airflow', 'tasks', 'run', 'dm1trybu', 'train_rf', 'scheduled__2023-10-21T09:45:00+00:00', '--job-id', '156', '--raw', '--subdir', 'DAGS_FOLDER/final_dag_3_models.py', '--cfg-path', '/tmp/tmpyd01z8t2']
[2023-10-22T22:06:10.276+0300] {standard_task_runner.py:85} INFO - Job 156: Subtask train_rf
[2023-10-22T22:06:10.428+0300] {task_command.py:415} INFO - Running <TaskInstance: dm1trybu.train_rf scheduled__2023-10-21T09:45:00+00:00 [running]> on host dsbulgako1-QEMU-Virtual-Machine
[2023-10-22T22:06:10.603+0300] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='dmitrii.bulghakov@bk.ru' AIRFLOW_CTX_DAG_OWNER='Dmitry Bulgakov' AIRFLOW_CTX_DAG_ID='dm1trybu' AIRFLOW_CTX_TASK_ID='train_rf' AIRFLOW_CTX_EXECUTION_DATE='2023-10-21T09:45:00+00:00' AIRFLOW_CTX_TRY_NUMBER='7' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-10-21T09:45:00+00:00'
[2023-10-22T22:06:10.634+0300] {s3.py:1359} INFO - This function shadows the 'download_file' method of S3 API, but it is not the same. If you want to use the original method from S3 API, please call 'S3Hook.get_conn().download_file()'
[2023-10-22T22:06:10.634+0300] {s3.py:1365} INFO - Downloading source S3 file from Bucket mlops-project with path dm1trybu/datasets/X_train.pkl
[2023-10-22T22:06:10.665+0300] {base_aws.py:553} WARNING - Unable to find AWS Connection ID 's3_connection', switching to empty.
[2023-10-22T22:06:10.665+0300] {base_aws.py:161} INFO - No connection ID provided. Fallback on boto3 credential strategy (region_name=None). See: https://boto3.amazonaws.com/v1/documentation/api/latest/guide/configuration.html
[2023-10-22T22:06:10.693+0300] {credentials.py:1254} INFO - Found credentials in shared credentials file: ~/.aws/credentials
[2023-10-22T22:06:11.062+0300] {configprovider.py:949} INFO - Found endpoint for s3 via: config_global.
[2023-10-22T22:06:11.945+0300] {s3.py:1359} INFO - This function shadows the 'download_file' method of S3 API, but it is not the same. If you want to use the original method from S3 API, please call 'S3Hook.get_conn().download_file()'
[2023-10-22T22:06:11.946+0300] {s3.py:1365} INFO - Downloading source S3 file from Bucket mlops-project with path dm1trybu/datasets/X_test.pkl
[2023-10-22T22:06:11.946+0300] {base_aws.py:161} INFO - No connection ID provided. Fallback on boto3 credential strategy (region_name=None). See: https://boto3.amazonaws.com/v1/documentation/api/latest/guide/configuration.html
[2023-10-22T22:06:11.959+0300] {credentials.py:1254} INFO - Found credentials in shared credentials file: ~/.aws/credentials
[2023-10-22T22:06:12.001+0300] {configprovider.py:949} INFO - Found endpoint for s3 via: config_global.
[2023-10-22T22:06:12.593+0300] {s3.py:1359} INFO - This function shadows the 'download_file' method of S3 API, but it is not the same. If you want to use the original method from S3 API, please call 'S3Hook.get_conn().download_file()'
[2023-10-22T22:06:12.593+0300] {s3.py:1365} INFO - Downloading source S3 file from Bucket mlops-project with path dm1trybu/datasets/y_train.pkl
[2023-10-22T22:06:12.593+0300] {base_aws.py:161} INFO - No connection ID provided. Fallback on boto3 credential strategy (region_name=None). See: https://boto3.amazonaws.com/v1/documentation/api/latest/guide/configuration.html
[2023-10-22T22:06:12.607+0300] {credentials.py:1254} INFO - Found credentials in shared credentials file: ~/.aws/credentials
[2023-10-22T22:06:12.643+0300] {configprovider.py:949} INFO - Found endpoint for s3 via: config_global.
[2023-10-22T22:06:13.138+0300] {s3.py:1359} INFO - This function shadows the 'download_file' method of S3 API, but it is not the same. If you want to use the original method from S3 API, please call 'S3Hook.get_conn().download_file()'
[2023-10-22T22:06:13.138+0300] {s3.py:1365} INFO - Downloading source S3 file from Bucket mlops-project with path dm1trybu/datasets/y_test.pkl
[2023-10-22T22:06:13.138+0300] {base_aws.py:161} INFO - No connection ID provided. Fallback on boto3 credential strategy (region_name=None). See: https://boto3.amazonaws.com/v1/documentation/api/latest/guide/configuration.html
[2023-10-22T22:06:13.151+0300] {credentials.py:1254} INFO - Found credentials in shared credentials file: ~/.aws/credentials
[2023-10-22T22:06:13.184+0300] {configprovider.py:949} INFO - Found endpoint for s3 via: config_global.
[2023-10-22T22:06:13.665+0300] {migration.py:213} INFO - Context impl PostgresqlImpl.
[2023-10-22T22:06:13.666+0300] {migration.py:216} INFO - Will assume transactional DDL.
[2023-10-22T22:06:28.496+0300] {logging_mixin.py:151} WARNING - /home/dsbulgako1/my_repos/mlops_project/env/lib/python3.10/site-packages/_distutils_hack/__init__.py:33 UserWarning: Setuptools is replacing distutils.
[2023-10-22T22:06:28.539+0300] {credentials.py:1254} INFO - Found credentials in shared credentials file: ~/.aws/credentials
[2023-10-22T22:06:55.862+0300] {python.py:194} INFO - Done. Returned value was: {'start_tiemstamp': '20231022 22:05', 'parent_run_name': 'parent_3_models', 'experiment_name': 'dm1trybu', 'experiment_id': '33', 'parent_run_id': '212ae9039a0b4dd8a51f6c3425799562', 'data_download_start': '20231022 22:05', 'data_download_end': '20231022 22:06', 'data_preparation_start': '20231022 22:06', 'data_preparation_end': '20231022 22:06', 'train_start_rf': '20231022 22:06', 'train_end_rf': '20231022 22:06'}
[2023-10-22T22:06:55.885+0300] {taskinstance.py:1398} INFO - Marking task as SUCCESS. dag_id=dm1trybu, task_id=train_rf, execution_date=20231021T094500, start_date=20231022T190610, end_date=20231022T190655
[2023-10-22T22:06:55.924+0300] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-10-22T22:06:55.949+0300] {taskinstance.py:2776} INFO - 1 downstream tasks scheduled from follow-on schedule check
